framework/src/main/java/org/checkerframework/framework/flow/CFAbstractTransfer.java

```diff
@@ -113,6 +113,9 @@ public abstract class CFAbstractTransfer<
   /** Indicates that the whole-program inference is on. */
   private final boolean infer;
 
+  /** Indicates that the resource leak checker is enabled. */
+  private final boolean enableWPIForRLC;
```

Comment:
I still don't think `CFAbstractTransfer` is the appropriate place for this flag. I agree that it's the LUB of `MustCallTransfer` and `CalledMethodsTransfer`, but this class should only contain code that's germane to all CF checkers.

Maybe create an interface with this functionality instead that those two classes can implement?